Gemfile.lock

```diff
@@ -150,7 +154,7 @@ DEPENDENCIES
   tzinfo-data
 
 RUBY VERSION
-   ruby 2.6.0p0
+   ruby 2.6.3p62
```

Comment:
@zaza81 I am not sure this is correct in the context of this PR.
In particular, I can see that:
https://github.com/italia/developers.italia.it/blob/95b72dd00853cb3b4aa2a1e43ef1ea06ba7cb84b/Dockerfile#L1
the Dockerfile explicitly uses `2.6.0`.
Is this needed for compatibility reason with the changes you are proposing?

Reply:
just for tracking last successful build. But I agree it's better to keep the Gemfile.lock generated from  `FROM circleci/ruby:2.6.0-node-browsers `, so I am reverting to ruby 2.6.0p0. 

Thanks for the review @libremente